controller-server/src/main/java/com/yahoo/vespa/hosted/controller/Application.java

```diff
@@ -51,14 +51,14 @@
     public Application(ApplicationId id) {
         this(id, DeploymentSpec.empty, ValidationOverrides.empty, Collections.emptyMap(),
              new DeploymentJobs(Optional.empty(), Collections.emptyList(), Optional.empty()),
-             Change.empty(), false, Optional.empty(), new ApplicationMetrics(0, 0),
+             Change.empty(), Change.empty(), Optional.empty(), new ApplicationMetrics(0, 0),
              Optional.empty());
     }
 
     /** Used from persistence layer: Do not use */
     public Application(ApplicationId id, DeploymentSpec deploymentSpec, ValidationOverrides validationOverrides,
                        List<Deployment> deployments, DeploymentJobs deploymentJobs, Change change,
-                       boolean outstandingChange, Optional<IssueId> ownershipIssueId, ApplicationMetrics metrics,
+                       Change outstandingChange, Optional<IssueId> ownershipIssueId, ApplicationMetrics metrics,
```

Comment:
can we rename the concepts _change_ and _outstandingChange_ to make it more clear that one is a vespa version change and the other an application change (given that my intepretation is correct ...)

Reply:
A change can be either a Vespa upgrade or an application version change. `outstandingChange` is just a change that has been postponed due to a change already being in progress.